docker/docker-compose-embedded.yaml

```diff
@@ -14,14 +17,25 @@ services:
       - 6002:6002
     depends_on:
       - ipfs
+      - lotus
     environment:
       - TEXPOWERGATE_EMBEDDED=true
+      - TEXPOWERGATE_LOTUSHOST=/dns4/lotus/tcp/7777
       - TEXPOWERGATE_IPFSAPIADDR=/dns4/ipfs/tcp/5001
-      - TEXPOWERGATE_LOTUSHOST=/dns4/lotus/tcp/1234
+    volumes:
+      - pow-embed-shared:/tmp/powergate
     restart: unless-stopped
 
   ipfs:
     image: ipfs/go-ipfs:latest
     ports:
       - 5001:5001
-    restart: unless-stopped    
+
+  lotus:
+    image: textile/lotus-devnet:lotus-testnet3-d41aeb
+    ports:
+      - 7777:7777
+    environment:
+      - TEXLOTUSDEVNET_SPEED=1500
+    volumes:
+     - pow-embed-shared:/tmp/powergate
```

Comment:
So, some thousands code lines less are resumed here.
The `textile/lotus-devnet` is running the devnet, exposing in 7777 the Lotus API.
Refer to that repo readme for the flags/envs possible.